src/app/admin/members/page.tsx

```diff
@@ -1,21 +1,20 @@
-import { publicPages } from "@/const/paths";
+"use client";
+// import { publicPages } from "@/const/paths";
 import { IndexTemplate } from "@/feature/admin/members/index";
-import { Seo } from "@/pages/_seo/Seo";
+// import { Seo } from "@/pages/_seo/Seo";
 import { FC } from "react";
 
 type Props = {};
 
-export const Index: FC<Props> = () => {
+export default function Index() {
```

Comment:
`export default function` succeded to build  pages

https://qiita.com/TaikiTkwkbysh/questions/e77a3b278ac73b691e70